binstar_client/utils/detect.py

```diff
@@ -76,6 +78,13 @@ def is_r(filename):
     else:
         log.debug("This not is an R package (expected .tgz, .tar.gz).")
 
+def is_cluster_profile(filename):
```

Comment:
I feel like we are gonna need more proof that this is a `cluster-profile` file. We are also gonna be uploading environments to AS and they also have `yaml` extension.
How about checking that it has a "provider", "node_id" and "number of nodes" something like that?